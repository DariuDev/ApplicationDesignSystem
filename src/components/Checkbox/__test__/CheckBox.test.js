// import React from 'react';
// import CheckBox from '../CheckBox';
// import Icon from 'react-native-vector-icons/MaterialIcons';
// import { fireEvent, render, wait } from '@testing-library/react-native';


// describe('CheckBox', () => {
//     jest.mock('react-native-vector-icons/MaterialIcons', () => 'Icon')
//   const activeCheckBox  = render(<CheckBox title='music' />);

//    const { getByTestId, }= activeButton ;
//   it('renders correctly', () => {
//     const activeCheckBoxAsJson = activeCheckBox.toJSON();
//     expect(activeCheckBoxAsJson).toMatchSnapshot();

//     // expect(getByText(caption)).toBeTruthy();
//     // fireEvent.press(getByTestId("custtomButton"));

//   });
// });


