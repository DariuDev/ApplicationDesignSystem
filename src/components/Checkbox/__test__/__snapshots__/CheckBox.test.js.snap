// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckBox shows the tickIcon when the checkBox checked 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "marginHorizontal": 5,
        "marginTop": 5,
        "width": 150,
      },
      Object {
        "justifyContent": "flex-end",
      },
    ]
  }
>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    testID="PressCheckBox"
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "justifyContent": "space-between",
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#000",
              "fontSize": 16,
              "fontWeight": "600",
              "marginLeft": 5,
            },
            Object {
              "color": "green",
            },
          ]
        }
      >
        music
      </Text>
      <Text
        allowFontScaling={false}
        selectable={false}
        style={
          Array [
            Object {
              "color": "red",
              "fontSize": 24,
            },
            undefined,
            Object {
              "fontFamily": "Material Icons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        î µ
      </Text>
    </View>
  </View>
</View>
`;
